docs/sql-migration-guide.md

```diff
@@ -36,6 +36,8 @@ license: |
 
   - In Spark 3.1, NULL elements of structures, arrays and maps are converted to "null" in casting them to strings. In Spark 3.0 or earlier, NULL elements are converted to empty strings. To restore the behavior before Spark 3.1, you can set `spark.sql.legacy.castComplexTypesToString.enabled` to `true`.
 
+  - In Spark 3.1, when `spark.sql.ansi.enabled` is false, Spark always returns null if the sum of decimal type column overflows. In Spark 3.0 or earlier, when `spark.sql.ansi.enabled` is false, the sum of decimal type column may return null or incorrect result, or even fails at runtime (depending on the actual query plan execution).
+
```

Comment:
nit: we need to describe `spark.sql.ansi.enabled is false` two times? I think its okay just to describe it like this;
```
In Spark 3.0 or earlier, the sum of...
```
or
```
In Spark 3.0 or earlier, in the case, the sum of...
```

Reply:
@maropu Thanks